ql-modem-helper.c

```diff
@@ -107,13 +111,15 @@ int main(int argc, char *argv[])
             {"get_fw_info", 0, NULL, 'G'},
             {"prepare_to_flash", 0, NULL, 'P'},
             {"flash_fw", 1, NULL, 'A'},
+            {"flash_mode_check", 0, NULL, 'M'},
             {"reboot", 0, NULL, 'R'},
-            {"sahara-reboot", 0, NULL, 'S'},
+            {"clear_attach_apn", 0, NULL, 'C'},
```

Comment:
Please add this option when we add the functionality to this option, currently this option is not doing any meaningful logic.

Reply:
> Please add this option when we add the functionality to this option, currently this option is not doing any meaningful logic.

modemfwd would flash carrier fw again if --clear_attach_apn return failure.